src/routes/Settings.test.tsx

```diff
@@ -250,4 +311,73 @@ describe('routes/Settings.tsx', () => {
     fireEvent.click(getByLabelText('Quit Gitify'));
     expect(ipcRenderer.send).toHaveBeenCalledWith('app-quit');
   });
+
+  it('should be able to enable colors', async () => {
+    let getByLabelText;
+    let findByLabelText;
+
+    jest.spyOn(apiRequests, 'apiRequestAuth').mockResolvedValue({
+      headers: {
+        'x-oauth-scopes': Constants.AUTH_SCOPE.join(', '),
+      },
+    } as unknown as AxiosResponse);
+
+    await act(async () => {
+      const {
+        getByLabelText: getByLabelTextLocal,
+        findByLabelText: findByLabelTextLocal,
+      } = render(
+        <AppContext.Provider
+          value={{
+            settings: mockSettings,
+            accounts: mockAccounts,
+            updateSetting,
+          }}
+        >
+          <MemoryRouter>
+            <SettingsRoute />
+          </MemoryRouter>
+        </AppContext.Provider>,
+      );
+      getByLabelText = getByLabelTextLocal;
+      findByLabelText = findByLabelTextLocal;
+    });
+
+    await findByLabelText('Use GitHub-like state colors');
+
+    fireEvent.click(getByLabelText('Use GitHub-like state colors'));
+
+    expect(updateSetting).toHaveBeenCalledTimes(1);
+    expect(updateSetting).toHaveBeenCalledWith('colors', true);
+  });
+
+  it('should not be able to disable colors', async () => {
```

Comment:
Ideally the `it` statement or context would communicate the circumstances under which enabling/disabling colors is allowed.

Also, I think this test might be intending to check that you cannot _enable_ colors?

Finally, I don't think the expectation is actually confirming that the option is disabled, right? Like, I think the expectation would also pass if the option is enabled.